srcpkgs/chromium/template

```diff
@@ -2,7 +2,7 @@
 pkgname=chromium
 # See http://www.chromium.org/developers/calendar for the latest version
 version=83.0.4103.97
-revision=1
+revision=2
```

Comment:
Don't increase the revision. There is no need to trigger the build servers later and block them for half a day.

Reply:
Done!